src/Microsoft.DotNet.ImageBuilder/src/Models/Annotations/OciManifest.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+#nullable enable
+namespace Microsoft.DotNet.ImageBuilder.Models.EolAnnotations
```

Comment:
Namespace doesn't match file path.
```suggestion
namespace Microsoft.DotNet.ImageBuilder.Models.Annotations
```

Reply:
Fixed with https://github.com/dotnet/docker-tools/pull/1334/commits/3227d5c9b5121df01f5059ed7dc3fec4e13d4950